### PR TITLE
Modifying Bullet constructor in Bullet, Alien and Spaceship class

### PR DESCRIPTION
**Pull request by Paola-Xaq**
Change constructor in Bullet class.
Change calling constructor in Spaceship and Alien class.

- (Its needed to) Review the Tests class in Spaceship, Alien. Need to change the constructor.

### CODE REVIEW

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Bullet.java [hunk 1]
```diff
@@ -42,34 +54,12 @@ public boolean getBulletExist() {
...
-        }
-    }
-
     /**
      * Description: The function displays a bullet shoot.
      * from the Spaceship.
      *
      */
-    public void isShootingToAlien(final Spaceship spaceship) {
+    public void isShootingToAlien() {
```
Comment (GustavoZHA) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/Bullet.java:62]: Rename the metod of isShootingToSpaceship

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Bullet.java [hunk 2]
```diff
@@ -81,7 +71,7 @@ public void isShootingToAlien(final Spaceship spaceship) {
      * from the Alien.
      *
      */
-    public void isShootingToSpaceship(final Alien alien) {
+    public void isShootingToSpaceship() {
```
Comment (GustavoZHA) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/Bullet.java:74]: A better name would be isShootingToAlien

**Review by danielaSantaCruz — APPROVED** [review]

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Bullet.java (hunk 2 above)
Comment (danielaSantaCruz) [reply src/main/java/org/fundacionjala/at13/spaceinvaders/Bullet.java:74]: The method is to say who "is shooting to Spaceship"? An Alien

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Bullet.java (hunk 1 above)
Comment (danielaSantaCruz) [reply src/main/java/org/fundacionjala/at13/spaceinvaders/Bullet.java:62]: The method is to say who "is shooting to Alien"? A Spaceship

**Review by JorgeCaceres-Jala — APPROVED** [review]

**Review by JessickaMoya — APPROVED** [review]

**Review by edsonarios — APPROVED** [review]

**Review by jpgonzales27 — APPROVED** [review]

**Review by cristianfchq — APPROVED** [review]

**Review by GustavoZHA — APPROVED** [review]

**Review by SaulCM — APPROVED** [review]

**Review by Raymundogs — APPROVED** [review]

**Pull request merged**